src/StreamJsonRpc/Reflection/MessageFormatterRpcMarshaledContextTracker.cs

```diff
@@ -396,7 +397,7 @@ internal MarshalToken GetToken(object marshaledObject, JsonRpcTargetOptions opti
     private static void ValidateMarshalableInterface(Type type, RpcMarshalableAttribute attribute)
     {
         // We only require marshalable interfaces to derive from IDisposable when they are not call-scoped.
-        if (!attribute.CallScopedLifetime && !typeof(IDisposable).IsAssignableFrom(type))
+        if (attribute.CallScopedLifetime && !typeof(IDisposable).IsAssignableFrom(type))
```

Comment:
This seems incorrect to me. And it seems unrelated to Nerdbank.MessagePack formatter support. Am I missing something?

Reply:
I don't think that was meant to be checked-in